lib/provisioning/Provisioner.js

```diff
@@ -365,20 +461,35 @@ Provisioner.prototype._setRequest = function (server, opNick, request) {
     this._pendingRequests[server.domain][opNick] = request;
 }
 
-Provisioner.prototype.handlePm = function(server, fromUser, text) {
+Provisioner.prototype.handlePm = Promise.coroutine(function*(server, fromUser, text) {
     if (['y', 'yes'].indexOf(text.trim().toLowerCase()) == -1) {
         log.warn(`Provisioner only handles text 'yes'/'y' ` +
                  `(from ${fromUser.nick} on ${server.domain})`);
+
+        this._sendToUser(
```

Comment:
We should probably yield here so handlePm only returns after we have sent this message.